stratum/hal/lib/barefoot/bf_sde_wrapper.cc

```diff
@@ -2967,11 +2971,24 @@ ::util::Status BfSdeWrapper::SynchronizeRegisters(
   return ::util::OkStatus();
 }
 
+std::string BfSdeWrapper::GetSdeVersionPrivate() const {
+#if defined(SDE_9_1_0)
+  return "9.1.0";
+#elif defined(SDE_9_2_0)
+  return "9.2.0";
+#elif defined(SDE_9_3_0)
+  return "9.3.0";
+#elif defined(SDE_9_3_1)
+  return "9.3.1";
```

Comment:
can we add `#error` for other cases that don't match the cases below

Reply:
I actually had an `#error` here before, but removed it because compilation will fail anyway (missing return on a non-void function). But an `#error` can be more clear with a message that gets printed.